### PR TITLE
fix tenant labels

### PR DESCRIPTION
**Pull request by klinch0**
## Summary by CodeRabbit

- **New Features**
	- Updated the HelmRelease configuration for monitoring agents to simplify tenant label assignment by using the release namespace directly.
  
- **Bug Fixes**
	- Adjusted the logging configuration for `fluent-bit` to ensure accurate categorization and processing of monitoring data.

### CODE REVIEW

File: packages/apps/kubernetes/templates/helmreleases/monitoring-agents.yaml [hunk 1]
```diff
@@ -45,7 +45,7 @@ spec:
     vmagent:
       externalLabels:
         cluster: {{ .Release.Name }}
-        tenant: {{ $targetTenant }}
+        tenant: {{ .Release.Namespace }}
```
Comment (kvaps) [inline packages/apps/kubernetes/templates/helmreleases/monitoring-agents.yaml:48]: Shouldn't it be `.Release.Name`?

Comment (klinch0) [reply packages/apps/kubernetes/templates/helmreleases/monitoring-agents.yaml:48]: no, release name - is name of HelmRelease (cluster name), like kubernetes-patch12toroot

Comment (klinch0) [reply packages/apps/kubernetes/templates/helmreleases/monitoring-agents.yaml:48]: tenant - its name of namespace, where HelmRelease

Comment (kvaps) [reply packages/apps/kubernetes/templates/helmreleases/monitoring-agents.yaml:48]: got it!

**Review by kvaps — APPROVED** [review]
LGTM

**Pull request merged**